### PR TITLE
Proposal to leave an application section

### PR DESCRIPTION
**Pull request by anedost**

### CODE REVIEW

File: src/css/common/section.css [hunk 1]
```diff
@@ -73,4 +73,15 @@
   .brand-underline::after {
     top: 31px;
   }
+}
+
+.visually-hidden {
```
Comment (VOSolyanik) [inline src/css/common/section.css]: already added

File: src/css/sections/proposal.css
Comment (VOSolyanik) [inline src/css/sections/proposal.css:1]: Unneccessary extra spacing and incorrect section size

As on designs
Screenshot 2024-10-04 at 01 00 40

As implemented
Screenshot 2024-10-04 at 01 00 05

File: src/css/sections/proposal.css [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+.proposal-container-info::before {
+  content: '';
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  border-radius: 16px;
+}
+
+.proposal-container-info::after {
+  top: 0px;
```
Comment (VOSolyanik) [inline src/css/sections/proposal.css]: top and left 0 is common and can be moved up into
```
.proposal-container-info::after,
.proposal-container-info::before {
```

File: src/partials/proposal.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      <div class="proposal-container-info">
+        <div class="proposal-container-info-main">
+          <p class="proposal-content-title">Power of English</p>
+          <div class="proposal-info-wrapper">
+            <div>
+              <ul class="proposal-content-list">
+                <li class="proposal-content-item">
+                  <div class="proposal-checkmark-icon">
+                    <svg class="check-icon">
+                      <use href="./img/sprite.svg#check"></use>
```
Comment (VOSolyanik) [inline src/partials/proposal.html]: ./images/sprite.svg
please merge main

File: src/partials/proposal.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    <div class="proposal-container-gradient">
+      <h3 class="proposal-section-heading">
+        Master the <span class="proposal-section-heading-color">Language</span>
+        with Our English Package!
+      </h3>
+      <div class="proposal-container-info">
+        <div class="proposal-container-info-main">
+          <p class="proposal-content-title">Power of English</p>
+          <div class="proposal-info-wrapper">
+            <div>
```
Comment (VOSolyanik) [inline src/partials/proposal.html:13]: unnecessary div

File: src/css/sections/proposal.css [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+  .proposal-container-info-main {
+    padding: 65px 0 63px 64px;
+  }
+
+  .proposal-content-list {
+    margin: 0 20px 39px 0;
+    max-width: 654px;
+  }
+
+  .books-photo {
```
Comment (VOSolyanik) [inline src/css/sections/proposal.css:199]: not needed as will work from 768

File: src/css/sections/proposal.css [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+    display: block;
+  }
+
+  .proposal-button {
```
Comment (VOSolyanik) [inline src/css/sections/proposal.css]: not needed as will work from 768

**Review by VOSolyanik — CHANGES_REQUESTED** [review]

File: src/partials/proposal.html (hunk 4 above)
Comment (anedost) [reply src/partials/proposal.html:13]: Screenshot 2024-10-04 at 10 14 49
Screenshot 2024-10-04 at 10 15 00

with and without this div

Comment (anedost) [reply src/partials/proposal.html:13]: @VOSolyanik

Comment (anedost) [reply src/partials/proposal.html:13]: this div it's are "merge solution" for list and button

Comment (anedost) [reply src/partials/proposal.html:13]: maybe you know best practices for this..
please commit or text answer
thx)

Comment (VOSolyanik) [reply src/partials/proposal.html:13]: oh, I see. Thanks

**Review by VOSolyanik — APPROVED** [review]

**Pull request merged**